whelktool/scripts/cleanups/2019/02/delete-weeded-holdings-for-Li/delete-weeded-holdings-for-Li.groovy

```diff
@@ -0,0 +1,26 @@
+/*
+ * This deletes all holdings for Li with availability 'UTGALLRAD'.
+ *
+ * See LXL-2275 for more info.
+ *
+ */
+String SIGEL_TO_DELETE = 'Li'
+String BIB_ID_FILE = 'lxl-2275-ids.txt'
```

Comment:
Where can I find the file with the id's? Missed to add to branch?

Reply:
Doh! Fixed now! :)